python/docs/source/reference/pyspark.sql/core_classes.rst

```diff
@@ -0,0 +1,37 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+============
+Core Classes
+============
+.. currentmodule:: pyspark.sql
+
+.. autosummary::
+    :toctree: api/
+
+    SparkSession
+    Catalog
+    DataFrame
+    Column
+    Observation
+    Row
+    GroupedData
+    PandasCogroupedOps
+    DataFrameNaFunctions
+    DataFrameStatFunctions
+    Window
```

Comment:
Can we have DataFrameReader DataFrameWriter?

Reply:
It's included in I/O, do we want to have separated category ?

Reply:
I meant the class name itself. The list here only contains methods.

Reply:
Got it. Just added